src/js/components/form.js

```diff
@@ -0,0 +1,64 @@
+export class Form {
+
+    API_LINK = "https://api.github.com/users/";
+    REPO_LINK = "/repos";
+    $userNameElement = document.getElementById('userName');
+    $userRepositoryListElement = document.getElementById('repoList');
+    $userAvatarElement = document.getElementById('userAvatar');
+
+    constructor() {
+        this.form = document.getElementById("main-form")
```

Comment:
jeśli wpisujesz jakies dane na sztywno, tzn nie sa one dynamicznie podawane przy tworzeniu instancji klasy, to nie potrzebnie je wrzucasz do constructora, constructor jest potrzebny tylko wtedy gdy te dane sa dynamicznie podawane :

const instancja = new Form(dane1, dane2, dane3);

jak ustawiasz je na sztywno, to rób to jako pola klasy, czyli tak jak ma w linijkach 3-7